package.json

```diff
@@ -21,8 +21,8 @@
   },
   "homepage": "https://github.com/redbadger/immutable-cursor",
   "dependencies": {
-    "atom-store": "0.0.2",
-    "immutable": "3.7.6"
+    "atom-store": "0.0.3",
```

Comment:
Am happy to publish a new release of `atom-store` at `0.0.3`, since the [change in argument order](https://github.com/jameshopkins/atom-store/commit/e809f638eb35566a38e72e4a5d8fb02630512ebd) has since been [reverted](https://github.com/jameshopkins/atom-store/commit/f9728c86eb7002b806b2f380459c4ec8d92bad1e).